### ISSUE TITLE
GoEmotions religion and name masking resources

### ISSUE THREAD

**Issue by blackcat84** [google-research/google-research#521]
Hello @ddemszky  @dattias and @eladeban,
I'm trying to apply your model to some text that is not part of the GoEmotions dataset.
To do so I need to mask the input text in the same way you do, that is replacing religions and names with the [RELIGION] and [NAME] tokens.
In the paper I read that the list of religion terms is included with the dataset, but I can't find it, maybe I'm looking in the wrong places?
Also, I'm looking for the BERT-based NER model you used for name tagging, do you have a link to the model by any chance?
Thank you very much!

**Comment by Alaa-1** [comment]
Same question ! Did you solve it ?

**Comment by blackcat84** [comment]
Nope, also asked for the BERT NER model directly to the authors no answer there as well. 
I've used my own list of religions to tag the religions and another NER model to tag the names ¯\_(ツ)_/¯

**Comment by ddemszky** [comment]
Thanks for these questions! For NER, we used the model described in the paper by [Tsai et al. (2019)](https://arxiv.org/abs/1909.00100). We used an internal version of it, but I found a public-facing Github [repository](https://github.com/DomHudson/bert-in-production), which seems to include this model, so you might find it useful. We'll get back to you regarding the other masked tokens. Hope this helps!

**Comment by Alaa-1** [comment]
> Thanks for these questions! For NER, we used the model described in the paper by [Tsai et al. (2019)](https://arxiv.org/abs/1909.00100). We used an internal version of it, but I found a public-facing Github [repository](https://github.com/DomHudson/bert-in-production), which seems to include this model, so you might find it useful. We'll get back to you regarding the other masked tokens. Hope this helps!

Thank you ^^ I'll check it out and hopefully get back to you.

**Comment by ddemszky** [comment]
An update regarding the masked tokens: we won't be able to release that list for now unfortunately, but you can recreate the tokens based on the Reddit comment ids. You can use diff to compare the comments in the dataset to the raw comments and you should be able to recreate those lists!

**Comment by Alaa-1** [comment]
@ddemszky This is rather a preliminary question: After getting the tags and making all of what you mentioned. Do I add the <person> and <religion> tokens as using ``` tokenizer.add_tokens(["NEW_TOKEN"]) ``` as discussed in here ( https://github.com/huggingface/transformers/issues/1413 ) , then train the model, or are there any pitfalls before getting to training?

 Thank you again for your time and effort ^^
